blogposts/2024-07-18-running-rnw-from-a-mac.md

```diff
@@ -0,0 +1,88 @@
+---
+post_title: "Building React Native for Windows Apps from a MacOS Device"
```

Comment:
Maybe it's because I've seen "device" so much, but... thoughts on updating the title?

```suggestion
post_title: "Building React Native for Windows Apps from a Mac"
```